API/authentication/app.js

```diff
@@ -98,6 +103,22 @@ async function createUser(username, password, email) {
  *
  */
 exports.lambdaHandler = async (event, context) => {
+  // Set database connection parameters, pulling credentials from env
+  const dbconnection = await mysql.createConnection({
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    user: process.env.DB_USER,
+    password: process.env.DB_PASS,
+    database: process.env.DB_DB,
+  });
+  // Connect to database
+  /*
```

Comment:
Thought this was deleted?

Reply:
Yeah I'm not sure why that's still in there. I'll remove it on the next go around.